Peppermint, Blood Red Dying, Cleanup and assorted things

Shnupbups opened the pull request:
- Added Peppermint
- Rebalanced the Trick or Treat loot table
- Switched the food stats of Rare Candy and Winged Strawberry Candy
- Added British English and Australian English translations
- Changed Spooktober command to say 'Spooktober 31st'
- Added translations for biomes that were missing them
- Reformatted code

Shnupbups committed:
Peppermint and assorted things

- Added Peppermint
- Rebalanced the Trick or Treat loot table
- Switched the food stats of Rare Candy and Winged Strawberry Candy
- Added British English and Australian English translations
- Changed Spooktober command to say 'Spooktober 31st'
- Added translations for biomes that were missing them
- Reformatted code

Shnupbups committed:
Merge branch 'master' into features

Juuxel commented on src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlockEntities.java:
```diff
@@ -0,0 +1,28 @@
+import java.util.function.Supplier;
+
+public class SpookyBlockEntities {
+	
```
Extra newline?

Juuxel commented on src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlockEntities.java:
```diff
@@ -0,0 +1,28 @@
+
+public class SpookyBlockEntities {
+	
+	public static final BlockEntityType<TinyPumpkinBlockEntity> TINY_PUMPKIN_BLOCK_ENTITY = register("tiny_pumpkin", TinyPumpkinBlockEntity::new, SpookyBlocks.TINY_PUMPKIN, SpookyBlocks.WITCHED_PUMPKIN);
```
We know it's a BE, so `TINY_PUMPKIN`.

Shnupbups committed:
Update SpookyBlockEntities.java

Shnupbups committed:
Merge branch 'features' of https://github.com/Shnupbups/spooky-time into features

Shnupbups committed:
Merge branch 'master' into features

Shnupbups committed:
Update TinyPumpkinBlockEntity.java

Shnupbups committed:
Blood Red Dyeing, more cleanup

- Throwing stuff into blood now dyes it red
- Abstracted the fluid crafting stuff
- Properly added fluid tags
- More cleanup

Shnupbups commented:
- Throwing stuff into blood now dyes it red
- Abstracted the fluid crafting stuff
- Properly added fluid tags
- More cleanup

Juuxel commented on src/main/java/com/fabriccommunity/spookytime/registry/SpookyTags.java:
```diff
@@ -16,4 +16,8 @@ private SpookyTags() {
 	}
+	
+	public static Tag<Item> register(String name) {
+		return new ItemTags.CachingTag(SpookyTime.id(name));
```
You can use Fabric's `TagRegistry.item` instead of `new ItemTags.CachingTag`.

Shnupbups committed:
Switched to TagRegistry for tags

Shnupbups committed:
Remove unused imports

vini2003 commented on src/main/java/com/fabriccommunity/spookytime/block/CraftingFluidBlock.java:
```diff
@@ -0,0 +1,69 @@
+import java.util.Map;
+
+public abstract class CraftingFluidBlock extends FluidBlock {
+	public final SoundEvent craftingSound;
```
`public static final SoundEvent CRAFTING_SOUND;` or death!

Shnupbups replied:
i can't make it static, it's defined in an initializer

Shnupbups replied:
so death i guess

vini2003 reviewed (changes requested):
Seems fine.
